lib/explorer/series.ex

```diff
@@ -27,10 +27,8 @@ defmodule Explorer.Series do
 
   @valid_dtypes Explorer.Shared.dtypes()
 
-  @type data :: Explorer.Backend.Series.t()
   @type dtype :: Explorer.Backend.Series.dtype()
-  @type t :: %Series{data: data, dtype: dtype()}
-  @type lazy :: %Series{data: Explorer.Backend.LazySeries.t(), dtype: dtype()}
```

Comment:
I think this type is being used by [`Explorer.Backend.Series`](https://github.com/elixir-nx/explorer/blob/0b446a7cb833aebe36514704340ba0ca295c4d7b/lib/explorer/backend/series.ex#L11)

Reply:
Tks, fixed!